src/main.c

```diff
@@ -306,15 +250,10 @@ static void neo_main(void) {
                         cx_sha256_t pubKeyHash;
                         cx_sha256_init(&pubKeyHash);
 
-                        cx_hash(&pubKeyHash.header, CX_LAST, publicKey.W, 65, result, 32);
-                        tx += cx_ecdsa_sign((void *) &privateKey,
```

Comment:
Do we really need to drop the signature here?

Reply:
Nope I'll put it back.